blog/2022/12-7-first-look-at-resoto-dashboards/index.md

```diff
@@ -0,0 +1,83 @@
+---
+authors: [pablo]
+tags: [ui, dashboard, visualization]
+---
+
+# First look at resoto dashboards
```

Comment:
let's update the title that tells the reader what to expect, and the benefit. something like:

"How to visualize your cloud inventory with Resoto Dashboards"
"How to create a custom cloud asset inventory with Resoto Dashboards"